packages/create-webstone-app/templates/plugin/structure/packages/web/README.md

```diff
@@ -0,0 +1,19 @@
+# XYZ - Web Webstone Plugin
+
+> **_Search & replace `XYZ` in this file with your plugin name._**
+
+## Installation
+
+Install this plugin with the following command:
+
+```
+npm install -D webstone-plugin-XYZ-web
+```
+
+## Usage
+
+> **_Document how developers can use this plugin. Do they import a Svelte component? Is there a Svelte action they can use?_**
+
+## Learn more about Webstone Plugins
+
+This plugin is part of a wider ecosystem called [Webstone Plugins](https://github.com/WebstoneHQ/webstone).
```

Comment:
I think this file will get overwritten by `create-svelte`. We would need to add a copy step afterward to get this radme

Reply:
Good catch, fixed in [34d2a80](https://github.com/WebstoneHQ/webstone/pull/310/commits/34d2a80f098cf51bee68adeb2826ce8e692bf939)